tibetegya/node-weconnect
# #162904902 add mongo db database

#### What does this PR do?
- add mongo db database
#### Description of Task to be completed?
- add mongoose package
- add database connection

#### How should this be manually tested?
- clone the repo git clone https://github.com/tibetegya/node-weconnect.git
- install the dependencies npm install
- make sure you have mongodb is installed, if not follow the instructions [here](https://docs.mongodb.com/v3.2/administration/install-community/)
- start the mongobd server by running `sudo services start mongodb`
- start dev server `npm run start:dev`
- using any http client navigate to `localhost:3000/`
- check the console
- if no error is thrown then mongodb has successfully connected else an error message is displayed
#### What are the relevant pivotal tracker stories?
[#162904902](https://www.pivotaltracker.com/story/show/162904902)
